Morphic.Settings/Registry/RegistrySettingHandler.cs

```diff
@@ -130,6 +130,10 @@ public static bool TryConvertToRegistry(object? value, RegistryValueKind registr
                 registryValue = null;
                 return false;
             }
+            if (value is int intValue)
```

Comment:
We might end up wanting this flexibility to take `int`s across all the setting handlers, but for now instead of changing just the registry handler let's instead just pass `long`s like we'd get from preferences.

Reply:
done